### PR TITLE
Collections ui

### PR DESCRIPTION
**Pull request by alchemistgo87**

### CODE REVIEW

File: src/app/(project)/projects/[project]/collections/[collection]/Collection.tsx [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+                </div>
+                <div className="flex items-center justify-between gap-2">
+                  <span className="text-gray-400">Created</span>
+                  <span>
+                    {format(convertLocalTime(collectionData?.createdAt), DateFormat.DATE_2)}
+                  </span>
+                </div>
+                <div className="flex items-center justify-between gap-2">
+                  <span className="text-gray-400">Authority</span>
+                  <span>{shorten(collectionData?.createdById)}</span>
```
Comment (kespinola) [inline src/app/(project)/projects/[project]/collections/[collection]/Collection.tsx:157]: This should be the update authority or the authority for polygon. We aren't saving that in hub-nfts right now so we should drop displaying this for now.

I'll notify @damianr

File: src/app/(project)/projects/[project]/collections/[collection]/nfts/Nfts.tsx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+              id: 'moreOptions',
+              header: () => <></>,
+              meta: {
+                align: 'right',
+              },
+              cell: (info) => {
+                return <></>;
+              },
+            }),
+          ]}
```
Comment (kespinola) [inline src/app/(project)/projects/[project]/collections/[collection]/nfts/Nfts.tsx:136]: Lets have the view on explorer for the nft as one of the actions.

File: src/app/(project)/projects/[project]/collections/page.tsx [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+              ]}
+              data={new Array(4)}
+            />
+          </>
+        ) : (
+          <>
+            <h1 className="text-2xl text-white font-medium">Collections</h1>
+            {noCollections ? (
+              <div className="h-full flex-1 flex flex-col items-center justify-center">
+                <Icon.Large.CreateProject />
```
Comment (kespinola) [inline src/app/(project)/projects/[project]/collections/page.tsx:116]: I don't think this is the write icon. Maybe set to the same one we use when there is no mints.

@damianr what should the empty state look like for collections?

**Review by kespinola — CHANGES_REQUESTED** [review]
Looks really good. Nice job.

File: src/app/(project)/projects/[project]/collections/page.tsx (hunk 3 above)
Comment (alchemistgo87) [reply src/app/(project)/projects/[project]/collections/page.tsx:116]: There is no specific icon for empty mints as well.

**Review by kespinola — APPROVED** [review]

**Pull request merged**